examples/server/tests/utils.py

```diff
@@ -89,6 +89,9 @@ def __init__(self):
             self.server_port = int(os.environ["PORT"])
 
     def start(self, timeout_seconds: int = 10) -> None:
+        if "LLAMA_SANITIZE" in os.environ:
+            timeout_seconds = 30
```

Comment:
Maybe doing this to limit `LLAMA_SANITIZE` to only control the default value:

```py
DEFAULT_HTTP_TIMEOUT = 10 if "LLAMA_SANITIZE" not in os.environ else 30
```

```suggestion
    def start(self, timeout_seconds_: int | None = DEFAULT_HTTP_TIMEOUT) -> None:
```